.circleci/config.yml

```diff
@@ -95,7 +96,7 @@ jobs:
       - run:
           name: Install Airflow chart
           command: |
-            pyenv global 3.8.5
+            pyenv global 3.9.1
```

Comment:
@kaxil @danielhoherd do you know if we are ok with python 3.9 instead of 3.8?

Reply:
This is just for CI.

Reply:
oh got it!